README.md

```diff
@@ -69,4 +84,15 @@ Build using Docker
 
 ```
 docker run -it -v $PWD:/scylla-machine-image -w /scylla-machine-image  --rm centos:7.2.1511 bash -c './dist/redhat/build_rpm.sh -t centos7 -c aws'
-```
\ No newline at end of file
+```
+
+## Building docs
+
+```bash
+python3 -m .venv
```

Comment:
I get 
```
$ python3 -m .venv
/usr/bin/python3: Relative module names not supported
```
with  Python 3.7.6

Reply:
sorry it's a typo...
it should be python3 -m venv .venv

Reply:
Also, I suggest to link to rst files from README
It will be easier on github users which does not want to build the docs.
Github render rst files (not as good, but still)

Reply:
@fruch FYI

Reply:
It won't render it with the plugin I've used
https://github.com/scylladb/scylla-machine-image/blob/next/source/user_data_v2.rst